pkg/gui/presentation/commits.go

```diff
@@ -307,6 +307,9 @@ func displayCommit(
 		color := lo.Ternary(commit.Action == models.ActionConflict, style.FgRed, style.FgYellow)
 		youAreHere := color.Sprintf("<-- %s ---", common.Tr.YouAreHere)
 		name = fmt.Sprintf("%s %s", youAreHere, name)
+	} else if isMarkedBaseCommit {
+		rebaseFromHere := style.FgYellow.Sprint(common.Tr.MarkedCommitMarker)
+		name = fmt.Sprintf("%s %s", rebaseFromHere, name)
```

Comment:
Marked the commit as "RFC" because this is not the most beautiful design. Suggestions for a better UI welcome; but I prefer this very explicit and in-your-face visualization over just coloring the sha.

Reply:
Another option would be to somehow darken the text of the commits from the marked commit down. Something like this (except for the background color, I'm not good with Photoshop 😅):
![Untitled](https://github.com/jesseduffield/lazygit/assets/1225667/8bd8cd59-f39e-40a0-9816-33e8306b8d16)

Unfortunately I have no idea how to achieve that, suggestions welcome.